ops/model.py

```diff
@@ -3421,6 +3422,19 @@ def _run(
             try:
                 result = subprocess.run(args, **kwargs)  # type: ignore
             except subprocess.CalledProcessError as e:
+                if 'access denied' in e.stderr.lower():
+                    if args[0] in ('juju-log', 'action-fail', 'action-set'):
```

Comment:
Can you clarify why the args are stripped for these specific commands?

Reply:
They all might contain sensitive data, so we should be cautious and not include the arguments. I've added a comment.